src/common/Utilities/Timer.h

```diff
@@ -10,6 +10,45 @@
 #include "Common.h"
 #include "Duration.h"
 
+enum class TimeFormat : uint8
+{
+    FullText,  // 1 Days 2 Hours 3 Minutes 4 Seconds 5 Milliseconds
+    ShortText, // 1d 2h 3m 4s 5ms
+    Numeric    // 1:2:3:4:5
+};
+
+enum class TimeOutput : uint8
+{
+    Days,         // 1d
+    Hours,        // 1d 2h
+    Minutes,      // 1d 2h 3m
+    Seconds,      // 1d 2h 3m 4s
+    Milliseconds, // 1d 2h 3m 4s 5ms
+    Microseconds  // 1d 2h 3m 4s 5ms 6us
```

Comment:
what purpose does microseconds have for us?

Reply:
This need for correct work few functions

Reply:
oh ok 👍